### ISSUE TITLE
Special Block font size inconsistent

### ISSUE THREAD

**Issue by hungryca-t** [redimp/otterwiki#136]
**Description**
The font size of special block (info, warning, etc.) content is inconsistent across paragraphs. Special blocks correctly display header text using header styling, and they correctly display content text for the first block of content using standard font styling. But subsequent paragraphs in the content section revert to header styling.

**Expected behavior**
Within special blocks, header text uses header styling while all subsequent text not denoted as a header uses standard font styling, including across paragraphs.

Markdown text, as well as preview screenshot, are provided here.

**Markdown**
```
# Test

:::info
# This is a header

This should be normal sized text.

This should be normal sized text.
:::

Here is another example with only one line of info text.

:::info
# This is a header

This should be normal sized text.
:::

:::warning
# Test with warning block

Test with warning block

Test with warning block
:::
```

![specialblock](https://github.com/user-attachments/assets/cde8bc49-8c87-40d0-b08e-8592b4ed9ec5)

**Comment by redimp** [comment]
Hey @hungryca-t, thank you for bringing this up!

Fix will be part of the next release. I'm not so happy with the design of the special blocks, just used the alerrts ofthe halfmoon framework.

Spoiler: Part of the next release will be Alerts in the github style:

```markdown
> [!NOTE]
> Useful information that users should know, even when skimming content.

> [!TIP]
> Helpful advice for doing things better or more easily.

> [!IMPORTANT]
> Key information users need to know to achieve their goal.

> [!WARNING]
> Urgent info that needs immediate user attention to avoid problems.

> [!CAUTION]
> Advises about risks or negative outcomes of certain actions.

```

![image](https://github.com/user-attachments/assets/8e8bf403-dc06-4fab-9d91-ea67b11bd006)

Have to make sure the font-sizes are correct in this one ^^

**Comment by redimp** [comment]
Released in [2.6.0](https://github.com/redimp/otterwiki/releases/tag/v2.6.0).
